src/components/v2/values/chartValuesDiff/ChartValuesView.type.ts

```diff
@@ -0,0 +1,156 @@
+import { OptionType } from '../../../app/types'
+import { ChartValuesType, ChartVersionType } from '../../../charts/charts.types'
+import { InstalledAppInfo, ReleaseInfo } from '../../../external-apps/ExternalAppService'
+import { ChartDeploymentDetail } from '../../chartDeploymentHistory/chartDeploymentHistory.service'
+
+export interface ChartValuesViewType {
+    appId?: string
+    isExternalApp?: boolean
+    isDeployChartView?: boolean
+    installedConfigFromParent?: any
```

Comment:
if possible please provide specific type rather than any

Reply:
Done.
Actually, installedConfigFromParent is being fetched & passed from 2 different APIs so keys may differ. So for now, created a type combining possible keys from both APIs.